packages/fxa-auth-server/lib/senders/emails/templates/test1/index.mjml

```diff
@@ -0,0 +1 @@
+<%- include('/partials/test/index.mjml') %>
```

Comment:
Hmm I'm not sure about having `test1` and `test2` templates in here... we do have one that's not an actual email, `_storybook`, that has that underline to denote that it's different than some of the others, but I think it would feel better for this to be in a separate directory, maybe even in `senders/emails/tests/`, but I'm wondering if we need these at all.  I think we need to just remove all dynamic references to l10n IDs like `data-l10n-id="<%- template %>-title` altogether. If we ever need to change the string, we have to also change the ID, making this really tricky to maintain. 😖 I haven't filed an issue for this yet, but I saw in the payments-server where we're also setting something dynamically through the ID and unfortunately that's going to break if we ever need to change the copy and update that ID (I can't find where I saw this to reference but I'll find it sometime soon).

I don't think we want extra `.ftl` files if possible too, since the l10n team is going to put in work to localize all of these strings and are going to have questions around what `{$testTerm}` is and might push back on this. If you'd like to leave all of this in anyway, maybe we can at least use less IDs and add a comment to let localizers know it's for dev testing purposes and they don't need translations.

Reply:
I also had concerns about the test1, test2 folders. I actually tried setting them as _test1, _test2, but then they would not get picked up for some reason. Possibly I can just piggyback off an existing template that is similar in nature for this PR, and side step issues surrounding these test folders.

I agree that we should not use the `data-l10n-id="<%- template %>-title` convention, it’s a bit too implicit for my liking. However, it is used quite a bit, and a motivation for this branch was fixing the issue without editing all the templates. I see phasing out the dynamic id as a secondary maintenance issue. After further testing, I see no technical reason why this shouldn’t work as is.

On a side note, I think there is future value in setting up a `senders/emails/tests` folder. There are two issues with our current tests. First they are repetitive, and second they may not fully exercise the logic in a partial template. Creating specific tests scenarios would allow our tests to be more targeted, specifically when it comes partials. 

I am not sure how I feel about your comments on language strings. I see your point, but I also think that ‘testing’ strings could be ignored by the translation team. Perhaps we should ask them? I do very much like your idea of not exporting the the `*.ftl` files if they are in the `senders/emails/tests` folder. I'd happy to address these things in this PR, even if it is beyond the scope of the initial issue.

Reply:
I think tests should go into the directory: `fxa-auth-server/test/local/senders/email`. I see where you are going with these email template tests and I am for it, but I don't think this is how it should be written - let's all regroup?